packages/admin/blocks-admin/src/blocks/factories/createOneOfBlock.tsx

```diff
@@ -374,13 +374,7 @@ export const createOneOfBlock = <T extends boolean = boolean>({
                                 {variant === "radio" && (
                                     <>
                                         <Box display="flex" flexDirection="column" padding={3}>
-                                            <FieldContainer>
-                                                {options.map((option) => (
-                                                    <Field key={option.value} name="blockType" type="radio" value={option.value} fullWidth>
-                                                        {(props) => <FormControlLabel label={option.label} control={<FinalFormRadio {...props} />} />}
-                                                    </Field>
-                                                ))}
-                                            </FieldContainer>
+                                            <RadioGroupField name="blocktype" fullWidth options={options} />
```

Comment:
Please make sure the name does not change

```suggestion
                                            <RadioGroupField name="blockType" fullWidth options={options} />
```